bin/setup.js

```diff
@@ -18,9 +18,7 @@ if (saveRequested) {
 
   if (packageJson.scripts[ 'start' ]) {
     log.info('setup', 'start script already set to "' + packageJson.scripts[ 'start' ] +
-    ', you can start hoodie with "npm run start-hoodie" instead')
-
-    packageJson.scripts[ 'start-hoodie' ] = 'hoodie'
```

Comment:
sorry but removing `start-hoodie` entirely is not what we meant. We still want to keep it if the `start` script is set to something like `ember server` for example.

What we want is to add another `if () {}` statement in which we check if the current value of `packageJson.scripts[ 'start' ]` starts with `'hoodie'`. If it does, don’t do anything. If it does not, then set `packageJson.scripts[ 'start-hoodie' ] = 'hoodie'`. Does that make sense?